travis_script.sh

```diff
@@ -7,7 +7,8 @@ declare -a  PROJECT_NAMES=(
 	"veggieseasons" \
 	"place_tracker" \
 	"platform_view_swift" \
-  "platform_design"
+    "isolate_example" \
```

Comment:
Check the spacing on these. I think the editor we quick-edited these in the other day played some games with the tabs and spaces.